crosschain-resolver/contracts/IMetadataResolver.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+interface IMetadataResolver {
+    /**
+     * @notice Get metadata about the CCIP Resolver
+     * @dev This function provides metadata about the CCIP Resolver, including its name, coin type, GraphQL URL, storage type, and encoded information.
+     * @param name The domain name in format (dnsEncoded)
+     * @return name The name of the resolver ("CCIP RESOLVER")
```

Comment:
What is this needed for?

Reply:
It's defined at https://docs.ens.domains/ens-improvement-proposals/ensip-16-offchain-metadata we can remove from the ENSIP if it's unnecessary

Reply:
I think we should probably do that.

Reply:
Removed